dev/documentation/en/utilities-jobs.md

```diff
@@ -0,0 +1,89 @@
+---
+book: dev-documentation
+version: 3.4
+title: Jobs - Technical Documentation - OJS|OMP|OPS
+---
+
+# Jobs
+
+Laravel [Queues](https://laravel.com/docs/9.x/queues) can be used to create, queue and process jobs. Use jobs to prevent long-running tasks from slowing down or freezing the application.
+
+## Create a Job
+
+Create a new class for each job.
+
+```php
+/**
+ * @file jobs/email/ExampleJob.php
+ */
+namespace PKP\jobs\email;
+
+use APP\submission\Submission;
+use PKP\jobs\BaseJob;
+
+class ExampleJob extends BaseJob
+{
+    protected Submission $submission;
+
+    public function __construct(Submission $submission)
+    {
+        parent::__construct();
+
+        $this->submission = $submission;
+    }
+
+    public function handle(): void
+    {
+        // Do something...
+    }
+}
+```
+
+## Dispatch a Job
+
+When a job is dispatched, it will be added to the job queue.
+
+```php
+use PKP\jobs\email\ExampleJob;
+
+dispatch(
+    new ExampleJob($submission)
+);
+```
+
+## Process Jobs
+
+The Admin Guide describes how to [configure a job runner in production](/admin-guide/en/configure#job-runner). For local development, one of the following methods may be preferred.
+
+1. Set the `job_runner` to `On` in the config file. Jobs will be run at the end of each web request.
+2. Run `php lib/pkp/tools/jobs.php run` from the command line to run all jobs.
+
+## Debug Jobs
+
+When a job fails, the message in the server's error log may not point to the failure. Use the CLI tool to view information about failed jobs.
+
+```
+php lib/pkp/tools/jobs.php failed
+```
+
+Failed jobs can also be found in the admin area of the application under Administration > Jobs > Failed Jobs.
+
```

Comment:
Also it is possible to do something or run some actions if a jobs failed to execute. To accomplish this, add the following method in job class with desired logic 
```php
/**
 * Handle a job failure.
 *
 * @param  \Throwable  $exception
 * @return void
 */
public function failed(\Throwable $exception)
{
    // Send user notification of failure or log etc ...
}
```

Reply:
Thanks, added at https://github.com/pkp/pkp-docs/pull/1063/commits/717a6ace518dc8d13ac1df2247baf6c905ae595a